docs/source/installation.rst

```diff
@@ -97,50 +97,70 @@ Linux OS
 ----------
 Create a virtual environment specific to this project:
 
-``conda create -n mesmerize python=3.9``
-``source activate mesmerize``
+.. code-block:: python
```

Comment:
this is bash not python, and it's easier to just put all the instructions in a single code block for copy-paste

Reply:
fixed